compiler/cgen.nim

```diff
@@ -1860,10 +1862,11 @@ proc myOpen(graph: ModuleGraph; module: PSym): PPassContext {.nosinks.} =
     incl g.generatedHeader.flags, isHeaderFile
 
 proc writeHeader(m: BModule) =
+  # xxx factor with 2 other instances
   var result = ("/* Generated by Nim Compiler v$1 */$N" &
-        "/*   (c) 2017 Andreas Rumpf */$N" &
+        "/*   (c) $2 Andreas Rumpf */$N" &
```

Comment:
Since I don't claim copyrights of the produced C code, instead remove this line altogether.

Reply:
right I was also wondering about that; what about the other 2 instances in `getCopyright`, shall i remove 
`"/*   (c) " & copyrightYear & " Andreas Rumpf */$N" &` from these as well?

Reply:
Yes.

Reply:
done